CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# How to contribute
+
+Add whatever it is that you would like to the README.md. You can fork the repo and edit locally, or simply use Github's built-in editor. 
+
+Try to follow existing formatting. In a lot of cases, it makes sense to alphabetize the items. If a list is already in alphabetical order, please add new items in alphabetical order as well.
```

Comment:
I'm generally in favor of that, I don't know about making it a hard rule as it might sometimes makes sense to "play favorites" in the list.